R/scope.R

```diff
@@ -1,22 +1,20 @@
 
 renv_scope_tempdir <- function(pattern = "renv-tempdir-",
                                tmpdir = tempdir(),
-                               envir = NULL)
+                               envir = parent.frame())
```

Comment:
For some reason, I was worried about `envir` being a promised and its value potentially depending on where / when that promise gets evaluated -- presumedly that's not actually an issue?

Reply:
That's what I thought you might be worried about, but no, it's not a problem.